src/utils/getTopics.js

```diff
@@ -0,0 +1,11 @@
+import axios from "axios"
+
+function getTopics() {
```

Comment:
I would hold all your api functions in one file, this isn't a hard rule, more of a recommendation